proknow-sdk/Requestor.cs

```diff
@@ -298,17 +298,26 @@ internal async Task<string> GetDomainStatusAsync()
             IList<KeyValuePair<string, string>> headerKeyValuePairs = null, HttpContent content = null)
         {
             string hasMore = "true";
-            string next = null;
+            string nextPage = null;
             var patients = new List<string>();
+            var patientsRoute = route.StartsWith("/collections") ? false : true;
```

Comment:
Check the whole route name and the HTTP method. This shouldn't be hard since this route doesn't have any variability (ie. no patient IDs). 

Also I prefer to prefix boolean/flag variables with either `is*`, `has*`, `are*`, etc.

```suggestion
            var isPatientsRoute = route == "/patients" && method == "GET";
```